scripts/build-worker.sh

```diff
@@ -54,6 +60,19 @@ pushd "$(dirname "$0")/../"
             +buildWorkerTypes "${WORKER_TYPE}" \
             +buildEnvironment "${BUILD_ENVIRONMENT}" \
             +scriptingBackend "${SCRIPTING_BACKEND}" \
+            "${TARGET_IOS_SDK_ARG}" \
             "${BUILD_TARGET_FILTER_ARG}"
+
+        if [[ "${BUILD_TARGET_FILTER-}" =~ "ios" ]]; then
+            echo "--- Building XCode Project :hammer_and_wrench:"
```

Comment:
```suggestion
            echo "--- Building XCode Project :xcode:"
```

Can't believe you wouldn't check for a custom emoji 😞

Reply:
:O How could I not have done that